pages/blog/blog.module.scss

```diff
@@ -0,0 +1,98 @@
+@import '../../styles/globals.scss';
+
```

Comment:
lets use our `$textBlack`  variable in this file for the font color so we're not using the default browser black